doc/cprover-manual/properties.md

```diff
@@ -147,6 +147,19 @@ unsigned foo(unsigned x)
   x = x + 2; // unsigned overflow checks are generated here
 ```
 
+#### Flag --nan-check limitations
+
+Please note that `--nan-check` flag is adding not-a-number checks only for
+generation of NaN value. Current implementation of `--nan-check` flag is not
+providing checks for propagation of NaN values.
```

Comment:
Clarify casts

Reply:
OK, I will add the following sentence:

> Generating assertions on type casting or structure/union member access is unsupported and such operation will not be examined.

so that the full text will look like:

> Please note that `--nan-check` flag is adding not-a-number checks only for generation of NaN value. Current implementation of `--nan-check` flag is not providing checks for propagation of NaN values. Generating assertions on type casting or structure/union member access is unsupported and such operation will not be examined.